polyfill/lib/absolute.mjs

```diff
@@ -0,0 +1,138 @@
+  const [one, two] = [this, other].sort(Absoulte.compare);
+  const delta = two.getEpochNanoseconds() - one.getEpochNanoseconds();
+  const nanos = Number(delta % 1000);
+  const micro = Number((delta / BigInt(1_000)) % 1000);
+  const milli = Number((delta / BigInt(1_000_000)) % 1000);
+  const secds = Number(delta / BigInt(1_000_000_000));
+  const Duration = ES.GetIntrinsic("%Temporal.Duration%");
+  return new Duration(0, 0, 0, 0, 0, secds, milli, micro, nanos);
+};
+Absolute.prototype.toString = function toString(timeZoneParam = "UTC") {
+  let timeZone = ES.ToTimeZone(timeZoneParam);
+  let dateTime = timeZone.getDateTimeFor(this);
+  let year = ES.ISOYearString(dateTime.year);
+  let month = ES.ISODateTimePartString(dateTime.month);
+  let day = ES.ISODateTimePartString(dateTime.month);
+  let hour = ES.ISODateTimePartString(dateTime.hour);
+  let minute = ES.ISODateTimePartString(dateTime.minute);
+  let seconds = ES.ISOSecondsString(
+    dateTime.second,
+    dateTime.millisecond,
+    dateTime.microsecond,
+    dateTime.nanosecond
+  );
+  let timeZoneString = ES.ISOTimeZoneString(timeZone, this);
+  let resultString = `${year}-${month}-${day}T${hour}:${minute}${
+    seconds ? `:${seconds}` : ""
+  }${timeZoneString}`;
+  return resultString;
+};
+Absolute.prototype.toLocaleString = function toLocaleString(...args) {
+  return new Intl.DateTimeFormat(...args).format(this);
+};
+Absolute.prototype.inZone = function inZone(timeZoneParam = "UTC") {
+  const timeZone = ES.ToTimeZone(timeZoneParam);
+  return timeZone.getDateTimeFor(this);
+};
+
+Absolute.fromEpochSeconds = function fromEpochSeconds(epochSecondsParam) {
+  let epochSeconds = ES.ToNumber(epochSecondsParam);
+  let epochSecondsBigInt = BigInt(epochSeconds);
+  let epochNanoSeconds = epochSecondsBigInt * BigInt(1_000_000_000);
+  let resultObject = new Absolute(epochNanoSeconds);
+  return resultObject;
+};
+Absolute.fromEpochMilliseconds = function fromEpochMilliseconds(
+  epochMillisecondsParam
+) {
+  let epochMilliseconds = ES.ToNumber(epochMillisecondsParam);
+  let epochMillisecondsBigInt = BigInt(epochMilliseconds);
+  let epochNanoSeconds = epochMillisecondsBigInt * BigInt(1_000_000);
+  let resultObject = new Absolute(epochNanoSeconds);
+  return resultObject;
+};
+Absolute.fromEpochMicroseconds = function fromEpochMicroseconds(
+  epochMicrosecondsParam
+) {
+  let epochMicroseconds = BigInt(epochMicrosecondsParam);
+  let epochNanoSeconds = epochMicroseconds * BigInt(1_000);
+  let resultObject = new Absolute(epochNanoSeconds);
+  return resultObject;
+};
+Absolute.fromEpochNanoseconds = function fromEpochNanoseconds(
+  epochNanosecondsParam
+) {
+  let epochNanoseconds = BigInt(epochNanosecondsParam);
+  let resultObject = new Absolute(epochNanoseconds);
+  return resultObject;
+};
+Absolute.fromString = function fromString(isoString) {};
+
+Object.defineProperty(Absolute.prototype, Symbol.toStringTag, {
+  get: () => "Temporal.Absolute"
```

Comment:
why are these getters? if they are getters, can they brand check? if not, why not data properties?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag

Reply:
Will change to value property